devise.rb

```diff
@@ -1,59 +1,50 @@
 run "if uname | grep -q 'Darwin'; then pgrep spring | xargs kill -9; fi"
 
-# GEMFILE
+# Gemfile
 ########################################
-inject_into_file 'Gemfile', before: 'group :development, :test do' do
+inject_into_file "Gemfile", before: "group :development, :test do" do
   <<~RUBY
-    gem 'devise'
-
-    gem 'autoprefixer-rails', '10.2.5'
-    gem 'font-awesome-sass'
-    gem 'simple_form', github: 'heartcombo/simple_form'
+    gem "devise"
+    gem "autoprefixer-rails"
+    gem "font-awesome-sass", "~> 6.1"
+    gem "simple_form", github: "heartcombo/simple_form"
   RUBY
 end
 
-inject_into_file 'Gemfile', after: 'group :development, :test do' do
-  <<-RUBY
-  gem 'pry-byebug'
-  gem 'pry-rails'
-  gem 'dotenv-rails'
-  RUBY
+inject_into_file "Gemfile", after: 'gem "debug", platforms: %i[mri mingw x64_mingw]' do
```

Comment:
```suggestion
inject_into_file "Gemfile", after: 'gem "debug", platforms: %i[ mri mingw x64_mingw ]' do
```

<img width="616" alt="Screenshot 2022-05-05 at 23 39 08" src="https://user-images.githubusercontent.com/6984619/167029975-2f45df85-8320-48a2-9ca2-a7720b8e5bfe.png">

Reply:
Ah it's my bad, I didn't know the space was necessary here, is it specific to symbols? Cause we don't need one for string arrays right?

Reply:
It's my fault :pray: 

I've removed the inner spaces here because from now on, I'm applying [standardrb](https://github.com/testdouble/standard) style in our content and inner spaces are not recommended inside arrays and hashes with this style.

But here, we are injecting some code after a line generated with `rails new` and the generated line has some inner spaces. So we need to match them.

Reply:
Thank you for the fix @db0sch :pray: 

@Annedj they are not mandatory: 

```irb
[1] pry(main)> %i[ mri mingw x64_mingw ]
=> [:mri, :mingw, :x64_mingw]
[2] pry(main)> %i[mri mingw x64_mingw]
=> [:mri, :mingw, :x64_mingw]
```

Reply:
Ahhh okay, didn't see that was the original line we were finding to inject new code, I thought we were injecting this one 🙈 Got it now 😄